cmd/qgb/deploy/cmd.go

```diff
@@ -0,0 +1,66 @@
+package deploy
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/celestiaorg/orchestrator-relayer/rpc"
+	"github.com/spf13/cobra"
+)
+
+func DeployCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "deploy <flags>",
+		Short: "Deploys the QGB contract and initializes it using the provided Celestia chain",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := parseDeployFlags(cmd)
+			if err != nil {
+				return err
+			}
+
+			// TODO add implementation
```

Comment:
linking for posterity https://github.com/celestiaorg/orchestrator-relayer/issues/49